packages/core/package.json

```diff
@@ -17,7 +17,8 @@
     "@salt-ds/icons": "workspace:^",
     "@salt-ds/styles": "workspace:*",
     "@salt-ds/window": "workspace:*",
-    "clsx": "^2.0.0"
+    "clsx": "^2.0.0",
+    "react-resizable-panels": "^2.1.7"
```

Comment:
@joshwooding  not everyone needs splitter but we are forcing everyone to install react-resizable-panels.
That feels different to @floating-ui which has multiple use-cases across components and everyone needs.
Should this be a separate package, or if not, what is the rationale, when to have a separate  package (date adapters) and when we can extend the core dependencies (react-resizable-panels) ?